components/automate-ui/src/styles.scss

```diff
@@ -234,3 +234,95 @@ input {
     border-radius: $global-radius $global-radius 0 0;
   }
 }
+
+// Angular Material Overrides
+.chef-control-menu-x {
+  position: absolute;
+  margin-top: 35px;
+  right: -48px;
+  margin-right: 35px;
+}
+
+.three-dot-column {
+  position: relative;
```

Comment:
Ah, nice! Since you have this, could you please remove the definition for `three-dot-column` occurring in all the other scss files (8 files). All of them are identical, containing a single line: `text-align: right;`. See if that is still needed; if so, should be able to add it here.

Reply:
FANTASTIC NOTE.  THANK YOU!  Deleting all that repetition was great